waveorder/models/phase2D_3D.py

```diff
@@ -0,0 +1,99 @@
+import torch
+import numpy as np
+from waveorder import optics, util
+
+
+def calc_TF(YX_shape, YX_ps, Z_pos_list, lamb_ill, n_media, NA_ill, NA_obj):
```

Comment:
Generally function names, parameters, and local variables should be snake_case according to [PEP8](https://peps.python.org/pep-0008/#function-and-variable-names).

Reply:
Also I wouldn't be too worried about having longer function names. Using complete words improve readability and the code can be auto-completed anyway.

Reply:
I've updated all of the names in the `phase3D_3D` and `phase2D_2D` code paths. I'll resolve this comment, but please feel free to continue to make suggestions.